draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -43,75 +43,37 @@ organization="Ping Identity"
 
 .# Abstract
 
-This specification defines a mechanism for selective disclosure of individual elements of a JSON object
-used as the payload of a JSON Web Signature (JWS) structure.
-It can be used for multiple applications, including but not limited to the selective disclosure of JSON Web Token (JWT) claims.
+This specification defines a mechanism for the selective disclosure of individual elements of a JSON-encoded data structure used as the payload of a JSON Web Signature (JWS).
```

Comment:
I'm curious to understand why "JSON-encoded data structure" is used here and elsewhere in the PR vs. "JSON object" or "JSON document" or "bit o' JSON" or something? It reads a bit off to my eye, at first anyway, but I wanted to ask before suggesting changes or otherwise fighting about it.

Reply:
Mainly to avoid "JSON object" as we can also SD the elements of an array.

Reply:
Ah ok. I'd read individual elements of a "JSON document" or "bit o' JSON" to mean all the stuff that might be inside that thing like objects or elements of an array. But I take your point or intention.